### ISSUE TITLE
Plotting date-times on y-axis, y-label obscured by toolbar

### ISSUE THREAD

**Issue by OwenArnold** [scipp/scipp#2176]
see https://github.com/scipp/scipp/pull/2169#issuecomment-915097903

**Comment by SimonHeybrock** [comment]
This is not specific to datetime or the y axis/labels. There are many cases where we have issues with this.

**Comment by OwenArnold** [comment]
> This is not specific to datetime or the y axis/labels. There are many cases where we have issues with this.
Yes, it seems to be associated with some of the padding defaults for matplotlib. I'll reword issue.
